.travis.yml

```diff
@@ -7,6 +7,13 @@ go:
   - 1.14
 os: osx
 
+addons:
+  homebrew:
+    packages:
+      - opencv
+      - pkgconfig
+    update: true
+
```

Comment:
Installing OpenCV in macOS CI is time-consuming. It takes about 25 minutes.

Reply:
I guess it's the brew install command who takes a long time to run, right?

If so, how about we build OpenCV for macOS manually from source code (https://docs.opencv.org/master/d0/db2/tutorial_macos_install.html), and upload it to AWS or a GitHub repo, so we could install it from our source without using Homebrew?

Reply:
I analyze the CI log, it takes about 5 min to update the Homebrew, and 15 min to install OpenCV.

OpenCV has a lots of dependencies, including: cmake, gflags, glog, metis, gmp, isl, mpfr, libmpc, gcc, openblas, gdbm, openssl@1.1, readline, sqlite, xz, python@3.8, tbb, suite-sparse, ceres-solver, aom, dav1d, freetype, frei0r, bdw-gc, libffi, libtool, pkg-config, guile, gettext, libtasn1, nettle, p11-kit, libevent, unbound, gnutls, lame, fribidi, pcre, glib, pixman, cairo, gobject-introspection, graphite2, icu4c, harfbuzz, libass, libbluray, libsoxr, libvidstab, libogg, libvorbis, libvpx, opencore-amr, jpeg, little-cms2, opus, rav1e, rtmpdump, flac, libsndfile, libsamplerate, rubberband, sdl2, snappy, speex, srt, webp, leptonica, tesseract, theora, x264, x265, xvid, ffmpeg, numpy, ilmbase, openexr, protobuf, boost, hdf5, netcdf, qt, sip, pyqt and vtk

I am afraid that we have to install lots of dependencies before building from source code. Let me have a try.